okapi-core/src/main/raml/TimerDescriptor.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "TimerDescriptor.json",
+  "title": "TimerDescriptor",
+  "description": "Timer",
+  "type": "object",
+  "additionalProperties" : false,
+  "properties": {
+    "id": {
+      "description": "Timer ID",
+      "type": "string"
+    },
+    "routingEntry": {
```

Comment:
Not sure  if the name `routingEntry` makes sense in a `TimeDescriptor`.

Reply:
What should it be called, then?  "entry" , "handler" ?